app/views/validation.scala.html

```diff
@@ -98,6 +98,9 @@
                             <u>N</u>ot sure
                         </button>
                     </div>
+                    <div id="svv-panorama-date-holder-0">
```

Comment:
I still don't think you need these lines, do you? This same code is on lines 84-86